src/main/java/com/mg105/Application.java

```diff
@@ -134,7 +135,7 @@ public void start(Stage primaryStage) {
         /////WinGame Scene/////
         ReplayGeneratorButton winButton = new ReplayGeneratorButton(replayGeneratorInterpreter, sceneController, Toggler.ToggleableComponent.WIN_MENU);
         WinMenu winMenu = new WinMenu(winButton);
-        WinDisplay winDisplay= new WinDisplay(sceneController, roomGetter);
+        WinDisplay winDisplay = new WinDisplay(sceneController, roomGetter, state);
```

Comment:
`state` is an entity and `WinDisplay` is an interface_adapter... so this is a clean architecture violation...